src/Providers/FeatureLayer.js

```diff
@@ -8,7 +8,7 @@ EsriLeafletGeocoding.Controls.Geosearch.Providers.FeatureLayer = L.esri.Services
     }
   },
   initialize: function(url, options){
-    L.esri.Services.FeatureLayer.prototype.call(this, url, options);
+    L.esri.Services.FeatureLayer.prototype.initialize.call(this, url, options);
```

Comment:
forgive me if i'm being dense, but what does this change in the FeatureLayer provider accomplish?